sklearn/calibration.py

```diff
@@ -41,9 +41,10 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin,
 
     Parameters
     ----------
-    base_estimator : instance BaseEstimator
+    base_estimator : BaseEstimator instance, default=None
```

Comment:
```suggestion
    base_estimator : estimator instance, default=None
```

I guess the existence of BaseEstimator is more an implementation detail

Reply:
@NicolasHug Fixed!